docs/overview.md

```diff
@@ -91,6 +78,6 @@ Ansible-NAS.
 The easiest way to take Ansible-NAS for a spin is in a virtual machine, for
 instance in [VirtualBox](https://www.virtualbox.org/). You'll want to create
 three virtual hard drives for testing: One of the actual NAS, and the two others
-to create a mirrored ZFS pool. Note because of the RAM requirements of ZFS,
-you might run into problems with a virtual machine, but this will let you
-experiment with installing, configuring, and running a complete system. 
+to create a mirrored ZFS pool. A virtual machine will probably not be happy or
```

Comment:
A virtual machine will be perfectly happy, it's just not recommended for "proper" use....because it's a VM. :)